examples/lfric/eg17/full_example_netcdf/README.md

```diff
@@ -64,36 +59,6 @@ steps are required for this (using simplified code examples):
     call invoke( name = 'testkern_w0', testkern_w0_type(field1, field2) )
     ```
 
-Significant refactoring was required in order to achieve that. This
-proof-of-concept implementation did not try a proper refactoring of
-code to reduce dependencies; in many case unnecessary dependencies
-were just removed from the source. Some examples:
-- XIOS was just removed (i.e. code that used it simply removed).
-- YAXT/MPI was completely removed (code that used it simply removed).
-- some dependencies were removed (for example the global_mesh module
-  also provides functionality for managing collections/maps of 
-  global meshes. This needlessly increase the number of files required).
-- Other support was just removed (e.g. NetCDF/UGRID, ...).
-
-A better designed stand-alone mini-example should
-- Refactor existing LFRic code to reduce dependencies
-  (e.g. instead of an object adding itself to a global data structure,
-  which makes the global handling a dependency for the object,
-  the global data structure can create a new object. This way
-  it is possible to create the object without dependencies).
-- For XIOS/YAXT NULL wrapper should be produced - i.e. they allow
-  compilation and linking, but offer no functionality. This removes
-  the need to remove significant amount of XIOS/YAXT code.
-- Other classes might be split into two classes/files. For example
-  there is no need that the unit-test functionality of a global
-  mesh is in the same file as the code that reads a mesh, allowing
-  the creation of a simple mesh by unit-testing code without
-  dependency to UGRID.
-
-Still, it is expected that there will be a porting effort involved
-when updating the stand-alone LFRic example from LFRic, but the goal
-is to minimise it, and automate it if possible.
-
 ## Compilation
 A simple makefile is provided to compile the example. It needs 
```

Comment:
Please insert a blank line between the header and the text.

Reply:
Done.